clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1476,6 +1476,10 @@ def NoDeref : DiagGroup<"noderef">;
 def BoundsSafetyCountedByEltTyUnknownSize :
   DiagGroup<"bounds-safety-counted-by-elt-type-unknown-size">;
 
+// counted_by warnings
+def CountedByAttribute
```

Comment:
`CountedByAttribute` seems misleadingly generic compared to `get-counted-by-side-effects`, which is very specific.